README.md

```diff
@@ -1,46 +1,37 @@
-# Electron Installer Grunt Plugin
+# Electron Installer
 
-[![Build status](https://ci.appveyor.com/api/projects/status/yd1ybqg3eq397i26/branch/master?svg=true)](https://ci.appveyor.com/project/kevinsawicki/grunt-electron-installer/branch/master)
+[![Build status](https://ci.appveyor.com/api/projects/status/yd1ybqg3eq397i26/branch/master?svg=true)](https://ci.appveyor.com/project/electronjs/windows-installer/branch/master)
```

Comment:
I just updated this on master and reconfigured the build, mind rebasing or merging the latest master and pushing up so that it will kick off a new appveyor build that will run the tests and everything.